src/core/issues-notes.js

```diff
@@ -245,7 +245,7 @@ function makeIssueSectionSummary(issueList) {
   ) {
     issueSummaryElement.insertAdjacentHTML(
       "afterbegin",
-      `<h2>${l10n.issue_summary}</h2>`
+      hyperHTML `<h2>${l10n.issue_summary}</h2>`
```

Comment:
This should be `` element.prepend(hyperHTML`...`) ``, because the current insertAdjacentHTML receives a string.

Reply:
```suggestion
      hyperHTML`<h2>${l10n.issue_summary}</h2>`
```

Our code style does not insert a space 😁

Reply:
@SuyashSalampuria, you can quickly fix these kinds of things with: `npm run lint -- --fix`

Reply:
@saschanaz Should I remove insertAdjacentHTML and use prepend there instead

Reply:
Using hyperHTML means we use elements instead, so yes we should use .prepend there.